### ISSUE TITLE
Develop the IA

### ISSUE THREAD

**Issue by HTD-CoCA** [cocaseattle/artist-directory#2]
Develop the Information Architecture of the Membership site after rating the Must Haves from 1-3.  Due Thursday 2/28

**Comment by Moirvae** [comment]
This is not doable by this week. It's dependent on #1 being done first, and agreed on.

**Comment by HTD-CoCA** [comment]
After we turn in our #1 on Thursday, we can collaborate on that same day and come to an agreement on that same day 2/28.  It could be later in the day but this needs to get done by the end of day Thursday 2/28.

**Comment by Moirvae** [comment]
Yes, we can agree on the top priority issues in a phone call, hopefully. But we can't develop an information architecture in a split second! It needs some thinking & trials.

**Comment by HTD-CoCA** [comment]
We can start with the fist draft and edit it as a live document until 3/4.  I can extend this until 3/4. How much time do you think you'll need?  The deadline for this should not be after Monday, 3/4.

**Comment by Moirvae** [comment]
It seems to me that there will be lots of other things to do, like finding coverage for the features. IA is very easy to implement in WP, but the actual structure shouldn't be rushed. I'm out of town 3/1 to 3/4, so will need the following week to consider what we agree on this week.
